src/utils/date.js

```diff
@@ -0,0 +1,14 @@
+import moment from "moment";
+
+const getWeekDays = todayDate =>
+  new Array(7).fill(null).map((v, dayOrderNumber) => {
+    return moment(todayDate)
+      .isoWeekday(dayOrderNumber + 1)
+      .format("DD.MM.YY dd");
```

Comment:
I recommend separating the presentation (view) concern from data generation concern.

This way, if you would need to additionally display a day date somewhere else (for example, in a modal on cell click), you would not need to generate another data object.

Overall, it's a slightly more effective approach to store data in a form, that is not dependent on how it will look like in UI. UI should get the original data and display it as it want.

Reply:
Ok, I moved ".format("DD.MM.YY dd")"  to the Table component